app/views/hyrax/base/_file_manager_attributes.html.erb

```diff
@@ -1,3 +1,11 @@
+<%# When navigating to the File Manager from a parent work, the `node`  %>
+<%# is a `WorkShowPresenter` instead of a `FileSetPresenter` which does %>
+<%# not have #extracted_text? which will cause a problem further down.  %>
+<%# To avoid an exception, we use the parent work's `FileSetPresenter`. %>
+<% if node.is_a?(Hyrax::WorkShowPresenter) %>
+  <% node = node.file_set_presenters.find { |fsp| node.file_set_ids.include?(fsp.id) } %>
+<% end %>
```

Comment:
Non-blocker, but wondering what would happen if you added `#extracted_text?` method to the `Hyrax::WorkShowPresenter`?  Yes, you'd probably need to make a `#file_set_presenter` method.

Reply:
That's a good question, the OCR lives on the file sets and not the work itself, so I'm wondering if we would still need to find it's file set anyways? 🤔